src/main/java/taskmanager/Storage.java

```diff
@@ -0,0 +1,74 @@
+package taskmanager;
+
+import tasks.Task;
+import tasks.Deadline;
+import tasks.Event;
+import exceptions.InvalidCommandException;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class Storage {
+
+
+    private List<Task> taskList;
+
+    public Storage() {
+        taskList = new ArrayList<Task>();
+    }
+
+    public void storageInsert(Task task) {
+        taskList.add(task);
+        int numOfTasks = taskList.size();
+
+        System.out.println("Got it. I've added this task:");
+        System.out.print(task.getStatusIcon() + " " + task.getContents());
+
+        if (task instanceof Deadline) {
+            System.out.print(" (by: " + ((Deadline) task).getDeadline() + ")");
+        } else if (task instanceof Event) {
+            System.out.print(" (from: " + ((Event) task).getStart() + " to: " + ((Event) task).getEnd() + ")");
```

Comment:
You can consider splitting this line to not reach the max character limit of 120.